README.md

```diff
@@ -135,6 +135,24 @@ userProfile.GetAttribute("age_over:18").Value.(string)
 
 GetAttribute returns an interface, the value can be acquired through a type assertion.
 
+
+You can retrieve the anchors, sources and verifiers for each attribute as follows:
+```Go
+givenNamesAnchors := userProfile.GivenNames().Anchors
+givenNamesSources := userProfile.GivenNames().Sources
+givenNamesVerifiers := userProfile.GivenNames().Verifiers
+```
+You can also retrieve further properties from these respective anchors in the following way:
+```Go
+givenNamesFirstAnchor := userProfile.GivenNames().Anchors[0]
+
+anchorType := givenNamesFirstAnchor.Type
+originServerCerts := givenNamesFirstAnchor.OriginServerCerts()
```

Comment:
1. No point in directing them to _originServerCerts_, because at the moment they can't do anything with that info.
2. We should offer some kind of explanation for what the _type_, _subType_ & _value_ are.  See the Java README

Reply:
So best to remove the accessor OriginServerCerts() for now too? And we can always add it in later?

Reply:
No, just drop it from the docs.  No harm in leaving access to the certs in place - but there's no point drawing attention to them when they can't be used for anything (yet)